utils/utils.py

```diff
@@ -56,7 +56,9 @@ def setup_ddp():
 
     except KeyError:
         print("DDP has to be initialized within a job - Running in sequential mode")
-
+    except RuntimeError:
```

Comment:
I'm a concerned about doing this, but it's the only way I could get the tests for all models to run right now

Reply:
Replaced with correct check for whether DDP has already been initialized